src/components/navigation/navItem/index.test.js

```diff
@@ -72,22 +61,12 @@ describe('Desktop version <NavItem/>', () => {
 
 describe('Mobile version <NavItem/>', () => {
   beforeAll(() => {
-    global.___loader = {
-      enqueue: jest.fn(),
-    }
-
     global.window.matchMedia = jest.fn(() => {
       return {
         matches: false,
-        addListener: jest.fn(),
-        removeListener: jest.fn(),
       }
     })
   })
```

Comment:
See comment above in regards to resetting the mock.

Reply:
Fixed!

Reply:
It is indeed, great work!